pkg/resolver/search.go

```diff
@@ -395,7 +410,18 @@ func getOperatorAndNumDateFilter(values []string) map[string][]string {
 				then = now.AddDate(-1, 0, 0).Format(format)
 
 			default:
-				operator = ""
+				if filter == "label" {
+					klog.V(7).Info("filter is label. Operator is @>.")
+					operator = "@>"
```

Comment:
We need to be able to discover which properties are a list.

In v1 we hardcoded the list of properties, which isn't ideal. [Here's the v1 implementation.](https://github.com/stolostron/search-api/blob/main/src/v2/connectors/redisGraph.js#L25-L27)

Reply:
Done.